src/lib.rs

```diff
@@ -54,68 +65,198 @@ type KVFilterList = Vec<(String, HashSet<String>)>;
 /// The filtering map can contain further key "packet" and value "send".
 /// With that the output for "receive" would be filtered.
 ///
-/// More precisely, a key is ignored until present and an entry must
-/// match for all the keys present any of the values to be left
-/// unfiltered. Refer to the test for an example.
+/// More precisely
```

Comment:
IIRC, markdown won't render the below as a list, until there's an empty line between.